frontend/packages/core/src/Contexts/app-context.tsx

```diff
@@ -2,9 +2,21 @@ import React from "react";
 
 import type { Workflow } from "../AppProvider/workflow";
 
+export type HeaderItems = "NPS";
```

Comment:
will this be a list of items in the future? just checking b/c the naming is a lil unclear

Reply:
I think this will end up being a union e.g. `"NPS" | "Foobar" but Josh can confirm

Reply:
Yep, @dschaller is correct, I can add a comment to clarify